commerce_coordinator/apps/iap/api/v1/views.py

```diff
@@ -0,0 +1,165 @@
+import json
+import logging
+
+from django.http import JsonResponse
+from django.views import View
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+
+from commercetools.platform.models.order import (
+    OrderFromCartDraft,
+    OrderUpdate,
+    OrderUpdateAction,
+    OrderSetLineItemCustomTypeAction,
+    OrderTransitionLineItemStateAction,
+)
+from commercetools.platform.models.payment import (
+    PaymentDraft,
+    PaymentMethodInfo,
+    PaymentStatusDraft,
+)
+from commercetools.platform.models.cart import (
+    CartAddPaymentAction,
+    CartSetShippingAddressAction,
+)
+from commercetools.platform.models.common import (
+    Address,
+    Money,
+    Reference,
+    ReferenceTypeId,
+)
+from commercetools.platform.models.type import TypeResourceIdentifier
+
+from commerce_coordinator.apps.commercetools.clients import CommercetoolsAPIClient
+
+logger = logging.getLogger(__name__)
+
+PAYPAL_PAYMENT_SERVICE_PROVIDER = "PayPal"
+
+@method_decorator(csrf_exempt, name='dispatch')
+class CreateOrderEndpoint(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            cart_id = data.get("cart_id")
+            cart_version = data.get("cart_version")
+            order_number = data.get("order_number")
+            payment_method = data.get("payment_method")
+            shipping_address_data = data.get("shipping_address")
+            paypal_order = data.get("paypal_order") or {
+                "id": "TEST12345",
+                "status": "APPROVED",
+                "paymentSource": {"type": payment_method or "paypal"}
+            }
```

Comment:
we can write a serlalizer for this validation. in this we, we don't have to manually handle the validation